src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/SortRewriter.cs

```diff
@@ -47,21 +47,30 @@ public override Expression VisitSqlRoot(SqlRootExpression expression, SqlSearchO
             // If the parameter being sorted on is part of a filter then we don't need to run the seperate search for resources that are missing a value for the field being sorted on.
             // If the parameter being sorted on is not part of a filter we need to run a seperate search to get resources that don't have a value for the field being sorted on.
             bool matchFound = false;
+            bool isSortHasMissingModifier = false;
             for (int i = 0; i < expression.SearchParamTableExpressions.Count; i++)
             {
                 Expression updatedExpression = expression.SearchParamTableExpressions[i].Predicate.AcceptVisitor(this, context);
+
+                if (expression.SearchParamTableExpressions[i].Predicate is MissingSearchParameterExpression)
+                {
+                    MissingSearchParameterExpression misingSPExpression = (MissingSearchParameterExpression)expression.SearchParamTableExpressions[i].Predicate;
+                    isSortHasMissingModifier = !misingSPExpression.IsMissing && context.Sort[0].searchParameterInfo.Name.Equals(misingSPExpression.Parameter.Name, System.StringComparison.OrdinalIgnoreCase);
```

Comment:
We are checking context.Sort[0] to see if it matches the missing parameter, is it possible to have other values in the Sort collection?  (Sort[1], Sort[2])?  Or it is always a collection of at most one item?
The FHIR spec does allow for multiple sort parameters, i.e.  <fhirServer>/Patient?_sort=name,birthdate

Reply:
When I specify more than one Sort, it ignore the sort and returns warning  " Only '_type' and '_lastUpdated' can be used together as sorting parameters (and in that order)".
When I specify one parameter, it honors that and add _lastUpdated as second sort param.